draft-ietf-quic-recovery.md

```diff
@@ -638,20 +638,21 @@ sooner, as soon as handshake keys are available (see Section 4.9.1 of
 
 # Congestion Control {#congestion-control}
 
-QUIC's congestion control is based on TCP NewReno {{?RFC6582}}.  NewReno is a
-congestion window based congestion control.  QUIC specifies the congestion
-window in bytes rather than packets due to finer control and the ease of
-appropriate byte counting {{?RFC3465}}.
+This document specifies a Reno congestion controller for QUIC {{?RFC6582}}.
 
-QUIC hosts MUST NOT send packets if they would increase bytes_in_flight (defined
-in {{vars-of-interest}}) beyond the available congestion window, unless the
-packet is a probe packet sent after a PTO timer expires, as described in
-{{pto}}.
+Endpoints can unilaterally choose a different algorithm to use, such as Cubic
```

Comment:
I think this could reads slightly  better if we swapped these two sentences. First state is generic - then that you can choose different alg.